alcs-api/src/card/card.service.ts

```diff
@@ -2,31 +2,99 @@ import { Mapper } from '@automapper/core';
 import { InjectMapper } from '@automapper/nestjs';
 import { Injectable, Logger } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
-import { Repository } from 'typeorm';
-import { CardUpdateDto } from './card.dto';
+import { FindOptionsRelations, Repository } from 'typeorm';
+import { Board } from '../board/board.entity';
+import { ServiceValidationException } from '../common/exceptions/base.exception';
+import { CardType } from './card-type/card-type.entity';
+import {
+  CardCreateDto,
+  CardDetailedDto,
+  CardDto,
+  CardUpdateServiceDto,
+} from './card.dto';
 import { Card } from './card.entity';
 
 @Injectable()
 export class CardService {
   private logger = new Logger(CardService.name);
+  private DEFAULT_RELATIONS: FindOptionsRelations<Card> = {
+    type: true,
+    status: true,
+    assignee: true,
+  };
 
   constructor(
     @InjectMapper() private mapper: Mapper,
     @InjectRepository(Card)
     private cardRepository: Repository<Card>,
+    @InjectRepository(CardType)
+    private cardTypeRepository: Repository<CardType>,
   ) {}
 
   get(uuid: string) {
-    return this.cardRepository.findOne({ where: { uuid } });
+    return this.cardRepository.findOne({
+      where: { uuid },
+      relations: {
+        ...this.DEFAULT_RELATIONS,
+        subtasks: {
+          type: true,
+          assignee: true,
+        },
+      },
+    });
   }
 
-  async update(cardUuid, card: Partial<CardUpdateDto>) {
+  async update(cardUuid, card: Partial<CardUpdateServiceDto>) {
```

Comment:
I think it would make more sense to add ? to the properties that have @isOptional on the DTO and not use Partial<> here